stdlib/dateBoundaries/dateBoundaries.flux

```diff
@@ -0,0 +1,417 @@
+    return _day_finder(td: date.Thursday, func: _day_formatter)
+}
+
+// friday will return starting and stopping point of the Friday this or last week
+// ## Parameters
+//
+//
+// ## Examples
+//
+// ### Find the Friday of that week
+//
+// ```no_run
+// import "dateBoundaries"
+// option location = timezone.fixed(offset: -8h)
+// option now = () => 2021-12-30T00:40:44Z
+// dateBoundaries.friday()
+//
+// Returns {start: 2021-12-24T08:00:00Z, stop:2022-12-25T08:00:00Z }
+// ```
+//
+// ```
+//
+// ### Filter by the records taken in a day
+// ```no_run
+// import "dateBoundaries"
+// ret = dateBoundaries.friday()
+// from(bucket: "example-bucket")
+//     |> range(start: ret.start, stop: ret.stop)
+// ```
+//
+// This will return all records from Friday this week
+//
+// ## Metadata
+// tags: date/time
+//
+friday = () => {
+    return _day_finder(td: date.Friday, func: _day_formatter)
+}
+
+// saturday will return starting and stopping point of the Saturday this or last week
+// ## Parameters
+//
+//
+// ## Examples
+//
+// ### Find the Saturday of that week
+//
+// ```no_run
+// import "dateBoundaries"
+// option location = timezone.fixed(offset: -8h)
+// option now = () => 2021-12-30T00:40:44Z
+// dateBoundaries.saturday()
+//
+// Returns {start: 2022-12-25T08:00:00Z, stop:2022-12-26T08:00:00Z }
+//
+//
+// ```
+//
+// ### Filter by the records taken in a day
+// ```no_run
+// import "dateBoundaries"
+// ret = dateBoundaries.friday()
+// from(bucket: "example-bucket")
+//     |> range(start: ret.start, stop: ret.stop)
+// ```
+//
+// This will return all records from Friday this week
+//
+// ## Metadata
+// tags: date/time
+//
+saturday = () => {
+    return _day_finder(td: date.Saturday, func: _day_formatter)
+}
+
+// sunday will return starting and stopping point of the Sunday this or last week
+// ## Parameters
+//
+//
+// ## Examples
+//
+// ### Find the Sunday of that week
+//
+// ```no_run
+// import "dateBoundaries"
+// option location = timezone.fixed(offset: -8h)
+// option now = () => 2021-12-30T00:40:44Z
+// dateBoundaries.sunday()
+//
+// Returns {start: 2021-12-26T08:00:00Z, stop:2021-12-27T08:00:00Z }
+//
+//
+//
+//
+// ### Filter by the records taken in a day
+// ```no_run
+// import "dateBoundaries"
+// ret = sunday()
+// from(bucket: "example-bucket")
+//     |> range(start: ret.start, stop: ret.stop)
+// ```
+//
+// This will return all records from Sunday this week
+//
+// ## Metadata
+// tags: date/time
+//
+sunday = () => {
+    return _day_finder(td: date.Sunday, func: _day_formatter)
+}
+
+// month will return the timestamps of the start and end of the month in a record
+// ## Parameters
+//
+//
+// ## Examples
+//
+// ### Find the timestamps for the start and stopping point of that month
+//
+// ```no_run
+// import "dateBoundaries"
+// option now = () => 2022-05-10T10:10:00Z
+// dateBoundaries.month()
+// // Returns {start:2022-05-01T00:00:00.000000000Z, stop:2022-06-01T00:00:00.000000000Z}
+// ```
```

Comment:
~~~suggestion
// ### Return start and stop timestamps for the current month
//
// ```no_run
// import "dateBoundaries"
//
// option now = () => 2022-05-10T10:10:00Z
//
// dateBoundaries.month()
// // Returns {start:2022-05-01T00:00:00.000000000Z, stop:2022-06-01T00:00:00.000000000Z}
// ```
~~~